framework/vst/CMakeLists.txt

```diff
@@ -0,0 +1,86 @@
+#=============================================================================
+#  MuseScore
+#  Music Composition & Notation
+#
+#  Copyright (C) 2020 MuseScore BVBA and others
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License version 2.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+#=============================================================================
+
+set(MODULE vst)
+
+if (NOT VST3_SDK_PATH)
+    message(FATAL_ERROR "VST3_SDK path is not setted")
+endif()
+
+if (CMAKE_BUILD_TYPE MATCHES "DEBUG")
+    add_compile_definitions(_DEBUG)
+else()
+    add_compile_definitions(NDEBUG)
+endif()
+
+set(MODULE_SRC
+    ${CMAKE_CURRENT_LIST_DIR}/vstmodule.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/vstmodule.h
+    ${CMAKE_CURRENT_LIST_DIR}/vstscaner.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/vstscaner.h
+    ${CMAKE_CURRENT_LIST_DIR}/pluginloader.h
+    ${CMAKE_CURRENT_LIST_DIR}/pluginloader.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/plugin.h
+    ${CMAKE_CURRENT_LIST_DIR}/plugin.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/plugininstance.h
+    ${CMAKE_CURRENT_LIST_DIR}/plugininstance.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/pluginparameter.h
+    ${CMAKE_CURRENT_LIST_DIR}/pluginparameter.cpp
```

Comment:
Better move files to internal, except vstmodule

Reply:
moved